node/node.go

```diff
@@ -190,15 +189,17 @@ func (n *Node) initNetworking() error {
 
 	tlsConfig := network.TLSConfig(n.Config.StakingTLSCert)
 
-	// Initialize validator manager and primary network's validator set
-	n.vdrs = validators.NewManager(n.Config.NetworkID)
-	networkValidators, ok := n.vdrs.GetValidators()
-	if !ok {
-		return errNoValidators
-	}
+	// Initialize empty validator set for now
+	n.validators = validation.NewSet()
+
+	// We add ourselves as validator with a weight of one here. This is a work-around
```

Comment:
Is this work-around permanent or we may have another solution in the future? In that case we should add some related info here.

Reply:
We won't have a P-chain in the future, so it won't be necessary then.